src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -1,3 +1,4 @@
+import { Link } from "@leafygreen-ui/typography";
```

Comment:
Use the spruce StyledLinks 

https://github.com/evergreen-ci/spruce/blob/cc67877ac01d9f5558213084a5c3e1fc6a1b0a05/src/components/styles/StyledLink.tsx